integration/pages/BasePage.js

```diff
@@ -0,0 +1,52 @@
+const config = require('../utils/config.js');
+const until = require('selenium-webdriver/lib/until');
+
+class BasePage {
+  constructor(driver) {
+    this.driver = driver;
+  }
+
+  async visit(route) {
+    await this.driver.get(config.baseUrl + route);
+  }
+
+  find(locator) {
+    return this.driver.findElement(locator);
```

Comment:
Isn't this an async function so find should be async too right?

Reply:
`this.driver.findElement(locator)` is NOT an async function,
`this.driver.findElements(locator)` is an async function.

It's weird but correct.

Reply:
From their main doc  https://www.selenium.dev/selenium/docs/api/javascript/module/selenium-webdriver/index_exports_WebDriver.html. You can see they use then.
![image](https://user-images.githubusercontent.com/58827450/143152695-0e1f61dc-014d-4004-b810-a23f354ad76f.png)

Reply:
Yeah, but if you check their doc and compare `findElement` and `findElements`:
https://www.selenium.dev/documentation/webdriver/web_element/#find-element
![image](https://user-images.githubusercontent.com/20471438/143153015-82e2c056-41c9-47b6-bd6d-343defb3181f.png)

Reply:
@ltrung check here, they are same:
https://www.selenium.dev/selenium/docs/api/javascript/module/selenium-webdriver/lib/webdriver_exports_WebElementPromise.html

So I think it's better to make it `async` to keep functions consistent. Will update later

Reply:
Changing the `findElement` to `async` breaks current test.

Taking clicking element as an example:
Now
```
// click() is async funciton
await driver.findElement({id: 'my-button'}).click();
```

After changing `findElement` to `async`, we have to update all our code to:
```
const myButton = await driver.findElement({id: 'my-button'});
await myButton.click();
//

or await (await driver.findElement({id: 'my-button'})).click();
```

There is no benefit in doing so.  Thus keep it as it is.